sdk/src/derivation_path.rs

```diff
@@ -1,3 +1,13 @@
+//! Definitions of standard Solana [BIP-44] key derivation paths.
```

Comment:
This module also handles non-Solana BIP-44 derivation paths. Should we allude to that in some fashion?

Reply:
I've modified this to indicate it works with general bip-44 paths.

This module needs more thorough docs, but not now.